api/proto/teleport/autoupdate/v1/autoupdate.proto

```diff
@@ -71,14 +71,17 @@ message AgentAutoUpdateSchedules {
 
 // AgentAutoUpdateGroup specifies the update schedule for a group of agents.
 message AgentAutoUpdateGroup {
+  reserved 4;
+  reserved "wait_days";
+
   // name of the group
   string name = 1;
   // days when the update can run. Supported values are "Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun" and "*"
   repeated string days = 2;
   // start_hour to initiate update
   int32 start_hour = 3;
-  // wait_days after last group succeeds before this group can run. This can only be used when the strategy is "halt-on-failure".
-  int64 wait_days = 4;
+  // wait_hours after last group succeeds before this group can run. This can only be used when the strategy is "halt-on-failure".
```

Comment:
Are zero or negative hours accepted? Is that worth mentioning?

Have you considered `google.protobuf.Duration wait_duration`?

Reply:
> Are zero or negative hours accepted? Is that worth mentioning?

No, I'll reject them during validation. Yes I'll add it.

> Have you considered google.protobuf.Duration wait_duration?

Yes but we do have a max resolution of an hour with `start_hour` so it makes more sense to keep the same unit.